src/components/pages/PendingApplications/PendingApplication.css

```diff
@@ -14,7 +14,11 @@
   border: none;
   color: white;
   padding: 0.5rem;
+<<<<<<< HEAD
```

Comment:
It looks like you somehow included an unresolved merge conflict message in your PR? Let's make sure to resolve this and then push those changes up so we don't have any production crashes!